.gitignore

```diff
@@ -137,3 +137,4 @@ results/
 
 # tests
 test_callback.log
+.mypy_cache/
```

Comment:
Can you remove this line from the Github commit? You can still keep it locally if you want.

Reply:
Whats the problem with including this in the gitignore?

Reply:
Good point! I thought it was better to remove as I wasn't sure whether it might add clutter or mess things in the repo (not sure if we are actually using mypy or not).